tests/sentry/workflow_engine/endpoints/test_organization_detector_workflow_index.py

```diff
@@ -178,6 +178,131 @@ def test_missing_body_params(self):
             self.organization.slug,
         )
 
+    def test_team_admin_create(self):
+        team_admin_user = self.create_user()
+        self.create_member(
+            team_roles=[(self.team, "admin")],
+            user=team_admin_user,
+            role="member",
+            organization=self.organization,
+        )
+        self.login_as(user=team_admin_user)
+
+        detector = self.create_detector(
+            project=self.create_project(organization=self.organization),
+            created_by_id=1,
```

Comment:
is the significance of this first part of the test that a team admin can edit a detector created by another user? it took me a bit to understand what was going on here, it'd be more clear if there was another user in the same org and team that were passed to this for creation and maybe add a comment since the other ones have one. Alternately this could be 3 separate tests

Reply:
To expand on this it might be useful to have more stuff in the setup method of this test class so you don't have to re-create the team admin user and member user

Reply:
yeah i guess this case could use a comment too, it's supposed to represent editing a user-created detector. i can also update this to use an actual user id